README.md

```diff
@@ -1,19 +1,34 @@
-# Kalix Java and Scala SDK
+# Kalix JVM SDKs
 
-For more information see the documentation for [implementing Kalix services in Java or Scala](https://docs.kalix.io/java/index.html).
 
-## Parts of the Kalix Java/Scala SDK
+The Kalix JVM SDKs are used to develop Kalix services using Java or Scala. Two different development approaches are available as listed below: 
+- Java SDK (code-first)
+- Java Protobuf SDK (protocol-first)
+- Scala Protobuf SDK (protocol-first)
 
-* user-facing packages
-    * `kalix-java-sdk` and `kalix-scala-sdk` The Java/Scala API to build services in Kalix (backed by an implementation in Scala based on Akka gRPC).
+If you’re just looking to get started, we advise you to start with the Java SDK which is the only one supporting a code-first development approach at present. For more information see the documentation for [implementing Kalix services in Java](https://docs.kalix.io/java/index.html).
 
-      See [Developing with Java or Scala](https://docs.kalix.io/java/index.html).
 
-    * `kalix-java-sdk-testkit` and `kalix-scala-sdk-testkit` A library to implement integration tests for services with Junit 4 or 5, based on [TestContainers](https://www.testcontainers.org/). Also contains library parts of unit TestKit.
+## Java SDK
 
-    * [samples](samples/) Small example services to illustrate Kalix features. The code provides snippets for the documentation.
+* User-facing packages
+  * `kalix-spring-boot-starter`: The Java API to build services in Kalix with a code-first development approach using a Spring Boot Starter. See [Java SDK](https://docs.kalix.io/java/index.html).
+  * `kalix-spring-boot-starter-test`: A library to implement integration tests for services with Junit 4 or 5, based on [TestContainers](https://www.testcontainers.org/). Also contains library parts of unit TestKit.
+* [Samples](samples/): Small example services to illustrate Kalix features for this SDK are prefixed with `java-spring-`. The code also provides snippets for the documentation.
+* Developer tooling
+  * `maven-java` Maven tooling
+    * `kalix-spring-boot-archetype` Maven archetype to create a project.
 
-* developer tooling
+## Java/Scala Protobuf SDK
+
+* User-facing packages
+  * `kalix-java-sdk-protobuf` and `kalix-scala-sdk-protobuf`: The Java/Scala Protobuf API to build services in Kalix (backed by an implementation in Scala based on Akka gRPC). See [Java/Scala Protobuf SDK](https://docs.kalix.io/java-protobuf/index.html).
```

Comment:
I would remove this part. I don't think it's relevant and some people may believe that they need to code in Scala.

```suggestion
  * `kalix-java-sdk-protobuf` and `kalix-scala-sdk-protobuf`: The Java/Scala Protobuf API to build services in Kalix. See [Java/Scala Protobuf SDK](https://docs.kalix.io/java-protobuf/index.html).
```